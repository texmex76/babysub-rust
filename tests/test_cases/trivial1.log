c BabySub Subsumption Preprocessor
c reading from '/home/gstrein/Documents/repos/babysub-rust/tests/test_cases/trivial1.cnf'
c parsed 'p cnf 1 1' header
c checked:                      0    clauses 0.00 per subsumed
c subsumed:                     0    clauses 0%
c process-time:                 0.00 seconds
