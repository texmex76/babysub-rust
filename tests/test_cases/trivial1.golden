c hash-signature 0
p cnf 1 0
